src/stylesheets/components/_hero.scss

```diff
@@ -27,7 +27,9 @@
   }
 
   > *:first-child {
-    @include margin(0 null units(2) null);
+    margin-bottom: units(2);
+    margin-top: 0;
+    @include u-margin-x(null);
```

Comment:
`null` just means "don't do anything" so this line can be removed. It's specific to that Bourbon mixin, I don't even know if it works in ours.

Reply:
yeah I already removed it, since it was failing my local builds. I just haven't pushed it yet